src/VecSim/spaces/space_aux.h

```diff
@@ -9,10 +9,17 @@
 
 enum Arch_Optimization {
     ARCH_OPT_NONE,
+#ifdef CPU_FEATURES_ARCH_X86_64
     ARCH_OPT_SSE,
     ARCH_OPT_AVX,
     ARCH_OPT_AVX512_F,
-    ARCH_OPT_AVX512_DQ
+    ARCH_OPT_AVX512_DQ,
+#endif
+#ifdef CPU_FEATURES_ARCH_AARCH64
+    ARCH_OPT_NEON,
+    ARCH_OPT_SVE,
+    ARCH_OPT_SVE2
```

Comment:
Let's add a comma at the end, to avoid future conflicts and the need to touch unrelated lines
```suggestion
    ARCH_OPT_SVE2,
```